command/certificate/verify.go

```diff
@@ -164,6 +178,35 @@ func verifyAction(ctx *cli.Context) error {
 		}
 	}
 
+	if validity {
+
+		remainingValidiy := time.Until(cert.NotAfter)
+		totalValidity := cert.NotAfter.Sub(cert.NotBefore)
+
+		percentUsed := ((totalValidity.Hours() - remainingValidity.Hours()) / totalValidity.Hours()) * 100
+
+		red := "\033[31m"
+		green := "\033[32m"
+		yellow := "\033[33m"
+		reset := "\033[0m"
+		
+		if percentUsed >= 100 {
+			fmt.Println(red, "This certificate has already expired.", reset)
```

Comment:
@tashian How would you like this output to look? Your plan was to use this as input from a pipe. I recall you saying that you wanted these to be errorCodes rather than printed statements.

Reply:
Yes, the error codes are described in #398